cmd/mayactl/app/command/volume_test.go

```diff
@@ -137,15 +155,17 @@ func TestGetControllerStatus(t *testing.T) {
 func TestGetIQN(t *testing.T) {
 	tests := map[string]struct {
 		expectedOutput string
-		Volume         CASVolume
+		Volume         VolumeInfo
 	}{
 		"Fetching IQN": {
-			Volume: CASVolume{
-				ObjectMeta: metav1.ObjectMeta{
-					Annotations: map[string]string{},
-				},
-				Spec: CASVolumeSpec{
-					Iqn: "iqn.2016-09.com.openebs.cstor:default-testclaim7",
+			Volume: VolumeInfo{
+				Volume: v1alpha1.CASVolume{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: map[string]string{},
+					},
+					Spec: v1alpha1.CASVolumeSpec{
+						Iqn: "iqn.2016-09.com.openebs.cstor:default-testclaim7",
```

Comment:
What if the spec is emtpy ? How do we handle it for 0.6 ?

Reply:
In 0.6 there is no spec so its fine

Reply:
But then ww need to find the value from annotation?
Cant it be like this that if not found in spec fetch from annotations?

Reply:
added to check for annotation if not present in spec

Reply:
That is great